features/mapping_history.feature

```diff
@@ -23,8 +23,8 @@ Feature: History of edits to a mapping
     And I visit the path /sites/directgov/mappings
     And I click the link "Edit"
     And I click the tab "History"
-    Then I should see that New URL was changed from http://somewhere.gov.uk to http://bad.gov.uk
-    And I should see "New URL updated"
+    Then I should see that New Url was changed from http://somewhere.gov.uk to http://bad.gov.uk
```

Comment:
Minor, these don't need changing, it's [captured by a regex](https://github.com/alphagov/transition/blob/be8f60aaa80efbed920daf629dc6f34b8e19b3a2/features/step_definitions/mappings_assertion_steps.rb#L39-L45) in the step definitions

Reply:
This was actually needed to correct a failing test, I was very surprised